config/zookeeper_persister.go

```diff
@@ -0,0 +1,161 @@
+// Licensed under the Apache License, Version 2.0
+// Details: https://raw.githubusercontent.com/maniksurtani/quotaservice/master/LICENSE
+
+package config
+
+import (
+	"bytes"
+	"errors"
+	"io"
+	"io/ioutil"
+	"sync"
+	"time"
+
+	"github.com/maniksurtani/quotaservice/logging"
+	"github.com/samuel/go-zookeeper/zk"
+)
+
+const (
+	SessionTimeout = 3 * time.Second
```

Comment:
Is that the golang consensus on consts? It feels weird to have a lowercase constant even if it is exported.